markdown/kubernetes_operator_codelab/kubernetes_operator_codelab.md

```diff
@@ -0,0 +1,243 @@
+
+
+
+
+<!--### Add a Link
+Add a link!
+[Example of a Link](https://www.google.com) -->
+
+<!--### Embed an iframe
+
+![https://codepen.io/tzoght/embed/yRNZaP](https://en.wikipedia.org/wiki/File:Example.jpg "Try Me Publisher")
+
+## Custom Step 1
+## Custom Step 2
+## Custom Step 3 -->
+
+## Environment Setup
+
+Duration: 0:03:00
+
+Below steps will help you create a kubernetes cluster in Kind and setup the environment for installing the Pubsub+ Event Broker Operator.
+
+### 1. Install Kind On macOS 
+#### Option A: Installing via Homebrew
+```bash
+brew install kind
+```
+#### Option B: Installing from Release Binaries
+```bash
+# for Intel Macs
+[ $(uname -m) = x86_64 ]&& curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.18.0/kind-darwin-amd64
+# for M1 / ARM Macs
+[ $(uname -m) = arm64 ] && curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.18.0/kind-darwin-arm64
+chmod +x ./kind
+mv ./kind /some-dir-in-your-PATH/kind
+```
+For other install options visit [Kind Installation](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+### 2. Create Kubernetes Cluster using Kind
+```bash
+kind create cluster
+```
+### 3. Verify that the Kubernetes Environment is ready
+```bash
+# This command returns the list of worker nodes and their status
+kubectl get nodes
+```
+![Environment Setup Verification](img/Operator_Img2.png)
+
+### Next Step is to install the Operator
+
+
+## Install Solace Pubsub+ Event Broker Operator
+
+Duration: 0:05:00
+
+An operator is a containerized application that works with the Kubernetes Operator Framework to deploy, manage and monitor other containerized applications in a declarative way.  The PubSub+ Operator deploys the PubSub+ Event broker in stand-alone or high-availability configurations. The [User Guide](https://github.com/SolaceProducts/pubsubplus-kubernetes-operator/blob/main/docs/EventBrokerOperatorUserGuide.md) for the PubSub+ Operator describes all of its capabilities.
+
+You can install the Operator in two ways:
+
+#### Option A: Operator Hub and OLM install
+Follow the steps from [OperatorHub](https://operatorhub.io/operator/pubsubplus-eventbroker-operator) to first setup OLM, then to install the PubSub+ Event Broker Operator. Click on the Install button to see the detailed instructions.
+#### Option B: Direct Install
+Today, you will learn to do the direct install.
+
+```bash
+# clone the directory
+git clone git@github.com:SolaceProducts/pubsubplus-kubernetes-operator.git
+
+# cd to the pubsubplus-kubernetes-operator/deploy folder and apply deploy.yaml
+# Manifest creates a namespace and all K8s resources for the Operator deployment
+kubectl apply -f deploy.yaml
+
+# Wait for deployment to complete
+kubectl get pods -n pubsubplus-operator-system --watch
+```
+Note: If you do not have SSH setup on your machine and got an error cloning the repo, you can clone the https link instead `https://github.com/SolaceProducts/pubsubplus-kubernetes-operator.git`
```

Comment:
I would suggest adding the same aside format here as before
`<aside class="positive"> If you do not have SSH .... </aside>`